pkg/ig-manager/ig-manager.go

```diff
@@ -26,7 +26,7 @@ import (
 	tracercollection "github.com/inspektor-gadget/inspektor-gadget/pkg/tracer-collection"
 )
 
-type LocalGadgetManager struct {
+type IgManager struct {
```

Comment:
This is the only one I'm not totally sure about. However I think for the time being we can use `IGManager` and then we can look for a better name. (once the reworking of kubectl-gadget being done by @flyth is finished).

Reply:
Yeah, It is strange. I agree we can go ahead with `IGManager` (and `igManager` as a variable) and then look for a better one after the refactoring is finished.